client/src/pages/HomePage/HomePage.js

```diff
@@ -1,114 +1,114 @@
+            fontSize: '32px',
+            lineHeight: '32px',
+            right: '47%',
+            bottom: '-25%',
+        },
+
+        [theme.breakpoints.up('desktop')]: {
+            right: '62%',
+        },
     },
-  },
 }));
 
-const UkraineText = styled((props) => <Typography variant="h1" {...props} />)(({ theme }) => ({
-  fontFamily: 'san serif',
-  margin: 0,
-  fontWeight: 700,
-  fontSize: '64px',
-  lineHeight: '64px',
-  textTransform: 'uppercase',
-  color: 'rgba(255, 255, 255, 0)',
-  WebkitTextStrokeWidth: '2px',
-  WebkitTextStrokeColor: 'rgb(41, 24, 194)',
-
-  [theme.breakpoints.up('tablet')]: {
-    fontSize: '96px',
-    lineHeight: '86px',
-    letterSpacing: '15px',
-  },
-
-  [theme.breakpoints.up('laptop')]: {
-    lineHeight: '117px',
-  },
+const UkraineText = styled((props) => <Typography variant="h1" {...props} />)(({theme}) => ({
+    fontFamily: 'san serif',
+    margin: 0,
+    fontWeight: 700,
+    fontSize: '64px',
+    lineHeight: '64px',
+    textTransform: 'uppercase',
+    color: 'rgba(255, 255, 255, 0)',
+    WebkitTextStrokeWidth: '2px',
+    WebkitTextStrokeColor: 'rgb(41, 24, 194)',
+
+    [theme.breakpoints.up('tablet')]: {
+        fontSize: '96px',
+        lineHeight: '86px',
+        letterSpacing: '15px',
+    },
+
+    [theme.breakpoints.up('laptop')]: {
+        lineHeight: '117px',
+    },
 }));
 
 const HeroContainer = styled(Container)({
-  position: 'absolute',
-  top: '50%',
-  left: '50%',
-  transform: 'translate(-50%, -50%)',
-  zIndex: 1000,
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    zIndex: 1000,
 });
 
 const HomePage = () => (
-  <>
-    <Box sx={{ position: 'relative' }} component="section">
-      <ImageCarousel />
-      <HeroContainer>
-        <TravelText>
-          TRAVEL <span className="heroHeaderHighlighted">around</span>
-        </TravelText>
-
-        <UkraineText>ukraine</UkraineText>
-
-        <Button sx={{ mt: '60px', textTransform: 'uppercase' }} endIcon={<ArrowRightAltIcon />}>
-          to our tours
-        </Button>
-      </HeroContainer>
-    </Box>
-    <Box component="section">
-      <AboutUkraine />
-    </Box>
-    <Box component="section">
-      <CardContainer />
-    </Box>
-  </>
+    <>
+        <Box sx={{position: 'relative'}} component="section">
+            <ImageCarousel/>
+            <HeroContainer>
+                <TravelText>
+                    TRAVEL <span className="heroHeaderHighlighted">around</span>
+                </TravelText>
+
+                <UkraineText>ukraine</UkraineText>
+
+                <Button sx={{mt: '60px', textTransform: 'uppercase'}} endIcon={<ArrowRightAltIcon/>}>
+                    to our tours
+                </Button>
+            </HeroContainer>
+        </Box>
+        <Box component="section">
+            <AboutUkraine/>
+        </Box>
+        <Box component="section">
+            {/*<CardContainer />*/}
```

Comment:
this is not corrent.. please leave it as it is as i already wrote the homepage test and mocked this component.